dask/test_dask.py

```diff
@@ -10,17 +10,21 @@ class DaskTestCase(DataprocTestCase):
     COMPONENT = 'dask'
     INIT_ACTIONS = ['dask/dask.sh']
 
+    SKEIN_TEST_SCRIPT = 'verify_skein.py'
     DASK_YARN_TEST_SCRIPT = 'verify_dask_yarn.py'
     DASK_STANDALONE_TEST_SCRIPT = 'verify_dask_standalone.py'
 
+    def verify_skein(self, name):
+        self._run_dask_test_script(name, self.SKEIN_TEST_SCRIPT)
+
     def verify_dask_yarn(self, name):
         self._run_dask_test_script(name, self.DASK_YARN_TEST_SCRIPT)
 
     def verify_dask_standalone(self, name):
```

Comment:
master_hostname to be passed as an argument.

Reply:
Does skein.Client() take a hostname as an argument?